### PR TITLE
#610 Completed updates to validation errors

### PR DESCRIPTION
**Pull request by GDonRanasinghe**
Satisfies #610

### CODE REVIEW

File: external-docs/docs/guides/errors/build-command-errors/multiple-observations-columns.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
-# Error - multiple observations columns defined
+# Error - multiple observations columns defined (Deprecated)
+
+## Deprecation warning
+
+This validation error will no longer be raised from csvcubed v0.1.18. This is because from v0.1.18, csvcubed supports pivoted multi-measure data sets which have more than one observations columns defined.
```
Comment (robons) [inline external-docs/docs/guides/errors/build-command-errors/multiple-observations-columns.md:5]: Could you tell them explicitly to upgrade the version of csvcubed they have installed - ideally giving them the conda/pip commands necessary.

File: tests/behaviour/qbwriter.feature [hunk 2]
```diff
@@ -652,8 +652,7 @@ Feature: Test outputting CSV-Ws with Qb flavouring.
 
 Scenario: A multi-measure pivoted shape cube should be produced as the output for the multi-measure pivoted shape inputs
   Given a multi-measure pivoted shape cube with identifier "qb-id-10003" named "Pivoted Shape Cube"
-  And the environment variable "PIVOTED_MULTI_MEASURE" is "True"
-  Then the CSVqb should pass all validations
+  When the CSVqb should pass all validations
```
Comment (robons) [inline tests/behaviour/qbwriter.feature:655]: I think this should be `Then` not `When`.

File: tests/unit/readers/cubeconfig/v1_0/test_build_friendly_error_msgs.py [hunk 3]
```diff
@@ -494,33 +456,6 @@ def test_val_errors_undefined_unit_uri():
     )
 
 
-def test_val_errors_uri_conflict():
```
Comment (robons) [inline tests/unit/readers/cubeconfig/v1_0/test_build_friendly_error_msgs.py:497]: Why has this one been deleted?

**Review by robons — COMMENTED** [review]
Just a few questions.

File: tests/unit/readers/cubeconfig/v1_0/test_build_friendly_error_msgs.py (hunk 3 above)
Comment (GDonRanasinghe) [reply tests/unit/readers/cubeconfig/v1_0/test_build_friendly_error_msgs.py:497]: This was deleted by mistake and added back again now.

**Review by robons — APPROVED** [review]

**Pull request merged**